service/javadsl/kafka/server/src/test/scala/com/lightbend/lagom/internal/javadsl/broker/kafka/JavadslKafkaApiSpec.scala

```diff
@@ -97,7 +97,10 @@ class JavadslKafkaApiSpec
         "lagom.cluster.join-self"                       -> "on",
         "lagom.cluster.exit-jvm-when-system-terminated" -> "off",
         "lagom.cluster.bootstrap.enabled"               -> "off",
-        "lagom.services.kafka_native"                   -> s"tcp://localhost:$kafkaPort"
+        "akka.discovery.method"                         -> "config",
+        "akka.discovery.config.services.kafka_native.endpoints" -> Seq(
+          Map("host" -> "localhost", "port" -> kafkaPort)
+        )
```

Comment:
Ah, I see...
Does this work with externally provided Kafka brokers, though? Is the `kafkaPort`  setting read when `lagomKafkaEnabled in ThisBuild := false`?
Also, hardcoding `"localhost"` will be limiting, even fo users running Kafka on their machine but binding to an external IP.

I'd rather stay with `DevModeServiceDiscovery` and the `sbt` overrides so `kafka_native` is looked up in `http://localhost:9008/services` by default.